### PR TITLE
Re-add AppStream

### PR DESCRIPTION
**Pull request by fenio**

### CODE REVIEW

**Comment by fenio** [comment]
I can't add reviewers myself.
@AsciiWolf can you add ximion here to allow him to confirm that such version is fine?
Also can you confirm you're good with such version?

File: gui/re.chiaki.Chiaki.appdata.xml [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+<component type="desktop">
+  <id>re.chiaki.Chiaki</id>
+  <launchable type="desktop-id">chiaki.desktop</launchable>
+  <name>Chiaki</name>
+  <summary>Free and Open Source Client for PlayStation 4 Remote Play</summary>
+  <metadata_license>CC0-1.0</metadata_license>
+  <project_license>GPL-3.0</project_license>
+  <developer_name>Florian Märkl</developer_name>
+  <url type="homepage">https://github.com/thestr4ng3r/chiaki</url>
+  <url typr="help">https://github.com/thestr4ng3r/chiaki/blob/master/README.md#usage</url>
```
Comment (AsciiWolf) [inline gui/re.chiaki.Chiaki.appdata.xml:12]: Typo: typ**r**

**Review by AsciiWolf — CHANGES_REQUESTED** [review]

**Comment by AsciiWolf** [comment]
LGTM, except the one typo in url tag. Just please squash the two commits into one and rename it to something better if possible. ;-)

**Comment by fenio** [comment]
I'm not sure how to squash two commits into one. Do I have to repeat everything? I don't see any option like that in GH.

**Comment by AsciiWolf** [comment]
You can use `git rebase -i HEAD~2` for this. For example, see [this](https://gist.github.com/jbub/5766366) guide. ;-)

File: gui/re.chiaki.Chiaki.appdata.xml [hunk 2]
```diff
@@ -0,0 +1,25 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!-- Copyright 2020 Florian Märkl <chiaki@metallic.software> -->
+<component type="desktop">
```
Comment (ximion) [inline gui/re.chiaki.Chiaki.appdata.xml:3]: Nitpick: `desktop-application` instead of `desktop` (but either will work fine)

**Review by ximion — APPROVED** [review]

**Comment by ximion** [comment]
Looks good to me! But please validate the metainfo file to find typos and other issues:
```
~$ appstreamcli validate /tmp/re.chiaki.Chiaki.appdata.xml
W: re.chiaki.Chiaki:12: url-invalid-type
E: re.chiaki.Chiaki:12: type-property-required url (https://github.com/thestr4ng3r/chiaki/blob/master/README.md#usage)

Validation failed: errors: 1, warnings: 1, pedantic: 2
```
(Personally I also would like more descriptive commit messages, but that's up to the project maintainer. Btw, you can leave out the `update_contact` field if you do not want to share your email address in cleartext - that field is rarely used, if at all, nowadays)

**Comment by AsciiWolf** [comment]
The warning from appstreamcli is the typo that I already pointed out. :-)

**Comment by ximion** [comment]
Yes, but it could have been found immediately if the validator was used ;-) You could also run this validation command automatically with the `--no-net` flag as a cmake test, to make sure the file is always valid. But that's a bonus, and completely not essential.

**Comment by AsciiWolf** [comment]
Yeah, that's true. :-) I personally always use AppStream validator. By the way, regarding the update_contact e-mail, that is an address provided by @thestr4ng3r.

**Comment by fenio** [comment]
Guys I *used* validator and that typo is fixed in 2nd commit which is also part of this PR.
I'm probably too dumb to do PR correctly as GH it still shows file with typo. Will try to fix it later today.

File: gui/re.chiaki.Chiaki.appdata.xml (hunk 1 above)
[the remark this answers is not in the document]
Comment (fenio) [reply gui/re.chiaki.Chiaki.appdata.xml]: Ok this should be fixed with rebase.

File: gui/re.chiaki.Chiaki.appdata.xml (hunk 2 above)
Comment (fenio) [reply gui/re.chiaki.Chiaki.appdata.xml:3]: I'd prefer not to introduce more changes in this PR.
It was desktop in previous version too. Let's keep it this way :)

**Comment by fenio** [comment]
Ok from my point of view this is something that can be merged. I know that comments of commits are pretty bad but I don't care. Changing them is not recommended as far as could read about it. I really wanted to simply reintegrate that stuff back. If all I prepared is not acceptable then well.. someone more familiar with git / github needs to do it.

**Comment by AsciiWolf** [comment]
The commits can be squashed during PR merge.

**Comment by thestr4ng3r** [comment]
@AsciiWolf would you like to approve the pr too?

**Review by AsciiWolf — APPROVED** [review]

**Comment by thestr4ng3r** [comment]
Thanks for the collaborative effort!

**Pull request merged**